__init__.py

```
@@ -31,14 +36,20 @@ def initialize(self):
-                           if not l.startswith("#") and l.strip()]
-                self.intent_matchers[lang].add_intent('launch', samples)
+            for intent_name in intents:
+                launch = join(self.root_dir, "locale", lang, f"{intent_name}.intent")
+                if not os.path.isfile(launch):
+                    continue
+                lang = standardize_lang_tag(lang)
+                if lang not in self.intent_matchers:
+                    self.intent_matchers[lang] = IntentContainer()
+                with open(launch) as f:
+                    # TODO - expand parentheses, do not require one per line
+                    samples = [l for l in f.read().split("\n")
```

Ruff

Ambiguous variable name: `l`

(E741)